Makefile

```diff
@@ -33,19 +33,13 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
-.PHONY: vendor
-vendor: ## Ensure the vendor directory is up to date.
-	go mod tidy
-	go mod vendor
-	go mod verify
-
 .PHONY: lint
 lint: ## Run golangci-lint over the codebase.
-	${GOLANGCI_LINT} run ./... --timeout 5m -v
+	${GOLANGCI_LINT} run ./... --timeout 5m -v ${GOLANGCI_LINT_EXTRA_ARGS}
 
 .PHONY: lint-fix
-lint-fix: ## Run golangci-lint over the codebase and run auto-fixers if supported by the linter
-	${GOLANGCI_LINT} run ./... --timeout 5m -v --fix
+lint-fix: GOLANGCI_LINT_EXTRA_ARGS := --fix
+lint-fix: lint ## Run golangci-lint over the codebase and run auto-fixers if supported by the linter
```

Comment:
```suggestion
lint-fix: l## Run golangci-lint over the codebase and run auto-fixers if supported by the linter
           GOLANGCI_LINT_EXTRA_ARGS := --fix $(MAKE) lint
```

Reply:
As far as I can tell, the way I have this set currently is the preferred make format for adding args and then depending on another target. This certainly works on my machine, have you tried this? Does it break for you?

Reply:
Oh sorry, it's fault on my configuration. It works fine after checkout this branch.